apps/push-service/src/push/configuration.ts

```diff
@@ -34,27 +53,10 @@ export const configurationSchema = {
             required: ['namespace', 'name'],
           },
         },
-        webhooks: {
-          type: 'array',
-          items: {
-            id: { type: 'string' },
-            name: { type: 'string' },
-            url: { type: 'string' },
-            targetId: { type: 'string' },
-            intervalMinutes: { type: 'number' },
-            description: { type: 'string' },
-            eventTypes: {
-              type: 'array',
-              items: {
-                id: { type: 'string' },
-              },
-            },
-          },
-        },
         subscriberRoles: { type: 'array', items: { type: 'string' } },
         publicSubscribe: { type: 'boolean' },
       },
-      required: ['id', 'name'],
+    
     },
```

Comment:
It looks we change the webhooks type. Do we need to delete the old data before deployment?